web/api/v1/api.go

```diff
@@ -372,14 +364,12 @@ func (api *API) query(r *http.Request) apiFuncResult {
 }
 
 func (api *API) queryRange(r *http.Request) apiFuncResult {
-	start, err := parseTime(r.FormValue("start"))
+	_, start, err := parseTimeParam(r, "start", minTime)
```

Comment:
I don't think we should be changing the behavior like this in a refactoring PR, without explicitly discussing / wanting it. Previously a missing start/end yielded an API error, now a default time is chosen. I'm not even sure if defaulting to minTime / maxTime is a good idea in general, because it will default to the most expensive query range (which could cause unintended overload for some people when they get API usage wrong).

Reply:
(for the `/api/v1/query_range` endpoint, that is)

Reply:
I thought is a good idea, but yeah, it doesn't make sense to query min-max range (except user wants it). I'll revert this back. Thanks!

Reply:
It looks like it is not reverted?